python_modules/dagster/dagster/_core/scheduler/instigation.py

```diff
@@ -564,6 +569,10 @@ class TickData(
             ("run_requests", Optional[Sequence[RunRequest]]),  # run requests created by the tick
             ("auto_materialize_evaluation_id", Optional[int]),
             ("reserved_run_ids", Optional[Sequence[str]]),
+            (
+                "user_interrupted",
+                bool,
+            ),  # indicates if a user stopped the tick while submitting runs
```

Comment:
so if the frontend wanted to see which work was requested but not executed, it could look at the run_requests that don't have their IDs included in the run_ids field?